src/snowflake/cli/plugins/nativeapp/run_processor.py

```diff
@@ -118,11 +117,9 @@ def ensure_app_usable(self, app: NativeAppProjectModel, show_app_row: dict):
         """Raise an exception if we cannot proceed with install given the pre-existing application object"""
 
         if self._requires_created_by_cli:
-            if show_app_row[COMMENT_COL] not in ALLOWED_SPECIAL_COMMENTS or (
```

Comment:
should we also add release notes on this?

Reply:
It's a relaxation of a case that the user is unlikely to get into. Part of it is covered by the release note (can now cross-upgrade to unversioned) but part of it is not (can now upgrade an unversioned app that was not created by SnowCLI). Happy to add that as a second release note if it makes sense!